tests/unit/QuickActionNavigationTest.ts

```diff
@@ -41,11 +49,30 @@ describe('IOU Utils', () => {
 
         it('should be navigated to Per Diem Expense', () => {
             // When the quick action is PER_DIEM
-            navigateToQuickAction(true, reportID, {action: CONST.QUICK_ACTIONS.PER_DIEM}, (onSelected: () => void) => {
+            navigateToQuickAction(true, {action: CONST.QUICK_ACTIONS.PER_DIEM, chatReportID: reportID}, {accountID: 1234}, undefined, (onSelected: () => void) => {
                 onSelected();
             });
             // Then we should start per diem request flow
             expect(startMoneyRequest).toHaveBeenCalledWith(CONST.IOU.TYPE.SUBMIT, reportID, CONST.IOU.REQUEST_TYPE.PER_DIEM, true);
         });
     });
 });
+
+describe('Non IOU quickActions test:', () => {
+    const reportID = generateReportID();
+
+    describe('navigateToQuickAction', () => {
+        it('creates new report for "createReport" quick action', () => {
+            navigateToQuickAction(true, {action: CONST.QUICK_ACTIONS.CREATE_REPORT, chatReportID: reportID}, {accountID: 1234}, undefined, (onSelected: () => void) => {
+                onSelected();
+            });
+            expect(createNewReport).toHaveBeenCalled();
+        });
+        it('starts create task flow for "assignTask" quick action', () => {
+            navigateToQuickAction(true, {action: CONST.QUICK_ACTIONS.ASSIGN_TASK, targetAccountID: 123}, {accountID: 1234}, undefined, (onSelected: () => void) => {
+                onSelected();
+            });
+            expect(startOutCreateTaskQuickAction).toHaveBeenCalled();
```

Comment:
I think we should add a test where we ensure the QAB data was correctly set in onyx